src/content/reference/cluster-upgrades/_index.md

```diff
@@ -64,14 +64,47 @@ Azure, AWS, and KVM based installations have independent versioning systems, as
 
 ### Determining the release version and inspecting release details
 
-As a user with access to the Giant Swarm API and being member of the organization owning the cluster, you can use both the web UI or the CLI to find out what release version your tenant cluster currently has.
+As a user with access to the Giant Swarm API, and being a member of the organization owning the cluster, you can use either the web UI or the CLI to find out the current release version of your tenant cluster.
 
 In the web UI, both the cluster overview and the cluster details page show the release version number. 
 In the cluster details page you can click the release version number to get more information about a release.
 
 Likewise in the CLI, commands like [`gsctl list clusters`](/reference/gsctl/list-clusters/) and [`gsctl show cluster`](/reference/gsctl/show-cluster/) reveal the release version number.
 To get information on all available releases, use the [`gsctl list releases`](/reference/gsctl/list-releases/) command.
 
+### Upgrage a cluster
+
+As an authenticated user you can upgrade the cluster version using the web UI or the CLI. The web UI shows a yellow link next to the version information if there is an upgrade available. For the CLI you can use the command [`gsctl upgrade cluster`](/reference/gsctl/upgrade-cluster/) for the same purpose.
+
+When the upgrade process is managed by our tools the release version chose is selected by Giant Swarm according to best practices. It means it will not upgrade a cluster by more than one major version at a time. In case you use the raw API to upgrade your cluster please test the process against a non-production cluster first.
+
+#### Cluster Upgrade Checklist
+
+Before triggering the upgrade, there are some checks we encourage you to perform:
+
+- Verify all your important pods are running correctly, and there no deployments stuck because of failures.
```

Comment:
```suggestion
- Verify all your important pods are running correctly, and there are no deployments stuck because of failures.
```

Sorry missed this.